_install-and-configure/install-opensearch/debian.md

```diff
@@ -0,0 +1,470 @@
+---
+layout: default
+title: Debian
+parent: Install OpenSearch
+nav_order: 55
+---
+
+{% comment %}
+The following liquid syntax declares a variable, major_version_mask, which is transformed into "N.x" where "N" is the major version number. This is required for proper versioning references to the Yum repo.
+{% endcomment %}
+{% assign version_parts = site.opensearch_major_minor_version | split: "." %}
+{% assign major_version_mask = version_parts[0] | append: ".x" %}
+
+# Debian
+
+Installing OpenSearch using the Advanced Packaging Tool (APT) package manager simplifies the process considerably compared to the [Tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/) method. Several technical considerations, such as the installation path, location of configuration files, and creation of a service managed by `systemd`, as examples, are handled automatically by the package manager.
+
+Generally speaking, installing OpenSearch from the Debian distribution can be broken down into a few steps:
+
+1. **Download and install OpenSearch.**
+   - Install manually from a Debian distribution or by creating a local repository.
```

Comment:
Since we are not actually creating the repo on the server, we just point the server to the remote repo on our url and say grab from there.

It might be better to say something like `Install manually from a Debian Package or from an APT repository`.